task_16/test/specs/main.js

```diff
@@ -0,0 +1,93 @@
+describe('Hard core', () => {
+    it('Open Google Cloud Homepage', async () => {
+        await browser.url('/')
+
+        //Find and click on search button
+        const searchBtn = await $('.devsite-searchbox > input')
+        await searchBtn.waitForExist({timeout: 5000})
+        await searchBtn.click()
+
+        //Type into search bar
+        await searchBtn.addValue('Google Cloud Platform Pricing Calculator')
+        await browser.keys("\uE007")
+
+        //Find result
+        const searchResult = await $('a.gs-title > b')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887245)
```suggestion
        const searchResult = await $("a.gs-title > b")
```